config/initializers/fast_gettext.rb

```diff
@@ -8,6 +8,11 @@
   gettext_options = %w(--sort-by-msgid --location --no-wrap)
   Rails.application.config.gettext_i18n_rails.msgmerge = gettext_options
   Rails.application.config.gettext_i18n_rails.xgettext = gettext_options + ["--add-comments=TRANSLATORS"]
+
+  config = VMDB::Config.new('vmdb').config
+  if config[:ui] && config[:ui][:mark_translated_strings] && !Rails.env.test?
```

Comment:
Slight simplifcation here, which prevents loading the config when it's not important (which is the majority of the time), as well as simpler to read by using fetch_path.

```ruby
if !Rails.env.test? && VMDB::Config.new('vmdb').config.fetch_path(:ui, :mark_translated_strings)
  include Vmdb::Gettext::Debug
end
```